docs/getting-started.md

```diff
@@ -116,57 +262,135 @@ That will let you start applications but you still can't interact with them: goo
+
+#### ACRN BOARD Configuration
+
+To build for your target board, set `ACRN_BOARD` in your `conf/local.conf`. By default it is set to `nuc7i7dnb`
+```
+ACRN_BOARD = "whl-ipc-i5"
 ```
 
-Now build the installer image:
+Supported Boards:
+- apl-mrb
+- apl-up2-n3350
+- apl-up2
+- nuc6cayh
+- nuc7i7dnb
+- whl-ipc-i5
+- whl-ipc-i7
+
+For More information, Please check [Supported Hardware](https://projectacrn.github.io/latest/reference/hardware.html)
 
+#### ACRN SCENARIO Configruation
+
+To build for your acrn scenario, set `ACRN_SCENARIO` in your `conf/local.conf`. By default it is set to `industry` scenario.
 ```
-$ bitbake mc:installer:acrn-image-base
+ACRN_SCENARIO  = "hybrid"
 ```
+Supported scenarios:
+- sdc
+- logical_partition
+- industry
+- hybrid
+- hybrid_rt
 
+For more information, please check [Build With the ACRN Scenario](https://projectacrn.github.io/latest/getting-started/building-from-source.html#build-with-the-acrn-scenario)
 
-### Override distro configuration
+To customize ACRN Configruation, please check [Introduction to ACRN Configuration](https://projectacrn.github.io/latest/tutorials/acrn_configuration_tool.html)
 
-Due to parsing sequence conflict with meta-intel, weak assingments are not used in acrn-demo-sos and acrn-demo-uos distros. So to override distro configuration in local.conf, override syntex can be used i.e
+#### ACRN BUILD MODE Configuration
 
-WKS_FILE_acrn-demo-sos = "your-custom.wks.in"
+To build ACRN in `RELEASE` mode, set `y` to `ACRN_RELEASE` in your `conf/local.conf`. By default it is set to `n`
+```
+ACRN_RELEASE = "y"
+```
+
+To build ACRN in `DEBUG` mode, set `n` to `ACRN_RELEASE` in your `conf/local.conf`.
+```
+ACRN_RELEASE = "n"
+```
+
+### Kernel Configuration
+
+There are multiple kernel variant available for both SOS and UOS.
 
+#### SOS
 
-### Optional GRUB Configuration Variables
+To switch to linux-intel-acrn-sos LTS 5.4 kernel (default), in 'local.conf' replace with below lines:
+```
+PREFERRED_PROVIDER_virtual/kernel = "linux-intel-acrn-sos"
+PREFERRED_VERSION_linux-intel-acrn-sos = "5.4%"
+```
 
-Below variables are used to generate grub.cfg. These variables can be overwrite in your local config.
+To switch to acrn-kernel-sos, in 'local.conf' replace with below lines:
+
+```
+PREFERRED_PROVIDER_virtual/kernel = "acrn-kernel-sos"
+```
+
+#### UOS
+
+
+To switch to linux-intel-acrn-uos LTS 5.4 kernel (default), in 'conf/multiconfig/uos.conf' replace with below lines:
+```
+PREFERRED_PROVIDER_virtual/kernel = "linux-intel-acrn-uos"
+PREFERRED_VERSION_linux-intel-acrn-uos = "5.4%"
+```
+
+To switch to linux-intel-rt-acrn-uos Preempt-RT 5.4 kernel, in 'conf/multiconfig/uos.conf' replace with below line:
+```
+PREFERRED_PROVIDER_virtual/kernel = "linux-intel-rt-acrn-uos"
+PREFERRED_VERSION_linux-intel-rt-acrn-uos = "5.4%"
+```
+
+To switch to acrn-kernel-uos, in 'conf/multiconfig/uos.conf' replace with below line:
+
+```
+PREFERRED_PROVIDER_virtual/kernel = "acrn-kernel-uos"
```

Comment:
Same question than for the sos kernel, what's the difference with the one previously mentionned?